.github/workflows/sync-testsite-branch.yml

```diff
@@ -0,0 +1,18 @@
+name: Sync Main to Storybook Site Branch
```

Comment:
I think this workflow needs to be merged into main so it runs on commits

Reply:
yep

Reply:
made a pr!: https://github.com/yext/search-ui-react/pull/273

Reply:
nice! in that case, can we delete the workflow changes from this PR? the syncing isn't needed on this branch technically, and it would be automatically merged in when the other PR gets merged into main

Reply:
yee. Removed changes!